task/gogit/README.md

```diff
@@ -0,0 +1,35 @@
+# Gogit
+
+This Task could send build status to GitHub/Gitlab.
```

Comment:
can you please point to the source code of gogit so that if somebody wants to build their own image then they can use that?

Reply:
also can you please elaborate more as which build status are we referring to?